generate/run.py

```diff
@@ -41,6 +43,22 @@ def find_clang_include_dir():
             return clang_include_path
 
 
+def find_vtk_include_dir():
+    for path in get_search_paths():
+        vtk_include_path = join(path, 'include', 'vtk-8.2')
+        if exists(vtk_include_path):
+            print("Found {}".format(vtk_include_path))
+            return vtk_include_path
+
+
+def find_tbb_include_dir():
+    for path in get_search_paths():
+        tbb_include_path = join(path, 'Library', 'include')
```

Comment:
I think it's the 'Library' here

Reply:
ya it needed that on windows, but prob not other platforms. just pushed a few more hacks, will try and rebuild and then assess